src/components/Accordion.astro

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
Why do we need this?

I'd like to aim for vanilla markdown and avoid complex components. Makes it easier to put a CMS on top later for anyone to edit

Reply:
Open to suggestions

Reply:
Where was that branch you were working on when you were doing your vibe coding to make each route its own page?